portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -410,6 +414,83 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 	}
 </script>
 
+<script data-senna-track="temporary" type="text/javascript">
+
+	var dropdown = document.getElementById("<%= namespace + id %>_ariaPagination");
```

Comment:
We still need this id to only be declared once. I think it's not working here because the ID is declared above inside of an if block. Really this script should also be defined inside the same if block so that if the html that this script renders is not on the page, then the script will also not be rendered.

Reply:
I see what you mean, I've moved the declaration to bottom at the page and now I can use it. Thanks Evan!